tools/verify_docker_gui.sh

```
@@ -0,0 +1,208 @@
+  exit 23
+fi
+
+if [[ -z "${FRAME_COMPARE_XAUTHORITY_PATH:-}" ]]; then
+  if [[ -n "${XAUTHORITY:-}" && -f "${XAUTHORITY}" ]]; then
+    export FRAME_COMPARE_XAUTHORITY_PATH="${XAUTHORITY}"
+  elif [[ -f "${HOME}/.Xauthority" ]]; then
+    export FRAME_COMPARE_XAUTHORITY_PATH="${HOME}/.Xauthority"
+  fi
+fi
+
+export FRAME_COMPARE_HOST_UID="$(id -u)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)